tests/ga/test_extension.py

```diff
@@ -376,185 +374,193 @@ def _set_up_update_test_and_update_gs(self, patch_command, *args):
         self._assert_ext_status(protocol.report_ext_status, "success", 0)
 
         # Next incarnation, update version
-        test_data.goal_state = test_data.goal_state.replace("<Incarnation>1<", "<Incarnation>2<")
-        test_data.ext_conf = test_data.ext_conf.replace('version="1.0.0"', 'version="1.0.1"')
-        test_data.manifest = test_data.manifest.replace('1.0.0', '1.0.1')
+        test_data.set_incarnation(2)
```

Comment:
Since i was touching those tests, i updated them to used the utility function i created a few PRs ago. This improves the readability (I may still rename some of those utilies)

Reply:
Thanks for this!